src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $background;
```

Comment:
The variable `$background` is used here, but it's not defined within this file. Ensure that it is defined in an imported file or within this file to avoid compilation errors.